packages/mdc-icon-button/index.ts

```diff
@@ -22,34 +22,25 @@
  */
 
 import MDCComponent from '@material/base/component';
-import MDCIconButtonToggleFoundation from './foundation';
 import {MDCRipple} from '@material/ripple/index';
+import MDCIconButtonToggleFoundation from './foundation';
 
-/**
- * @extends {MDCComponent<!MDCIconButtonToggleFoundation>}
- */
-class MDCIconButtonToggle extends MDCComponent {
-  static attachTo(root) {
+class MDCIconButtonToggle extends MDCComponent<MDCIconButtonToggleFoundation> {
+  static attachTo(root: Element) {
     return new MDCIconButtonToggle(root);
   }
 
+  private ripple_: MDCRipple;
+  // TODO: type for event handler?
+  // tslint:disable-next-line:no-empty
+  private handleClick_ = (() => {});
```

Comment:
try this: 
```

import {SpecificEventListener} from '@material/dom/index';

...
private handleClick_: SpecificEventListener<'click'>;
```

Reply:
that didn't work. used `EventListenerOrEventListenerObject` type instead.